src/processes/helper/index.js

```diff
@@ -11,6 +11,7 @@ const followMainFeedTopic = require('./followMainFeedTopic');
 const followMainFeedFollowing = require('./followMainFeedFollowing');
 const followMainFeedF2 = require('./followMainFeedF2');
 const unFollowMainFeedF2 = require('./unFollowMainFeedF2');
+const sendTopicAutoMessage = require('./sendTopicAutoMessage');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `sendTopicAutoMessage` is correctly added. However, it would be beneficial to maintain consistency in the codebase by adding a comment describing what this module does, similar to other imports.
```diff
+ // Module to send automatic messages during topic registration
  const sendTopicAutoMessage = require('./sendTopicAutoMessage');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->